ts/features/bonus/cgn/store/reducers/activation.ts

```diff
@@ -0,0 +1,47 @@
+// bonus reducer
+import { getType } from "typesafe-actions";
+import { Action } from "../../../../../store/actions/types";
+import { cgnActivationStatus } from "../actions/activation";
+
+export enum CgnActivationProgressEnum {
+  "UNDEFINED" = "UNDEFINED",
```

Comment:
Since the reducer use only these states: `SUCCESS`, `PROGRESS`, `UNDEFINED`, `ERROR`
the other states could be removed? (if we'll need them, we can add them later)

or

You can replace these state with [RemoteValue](https://github.com/pagopa/io-app/blob/214f241851674d762a6c63e1a9a7ff9db06ab965/ts/features/bonus/bpd/model/RemoteValue.ts)?

Reply:
We're going to use all these values cause we need to request the activation status with a polling as we did with Bonus Vacanze, so even the PENDING/EXISTS values may be triggered.

The reducer will be evaluated properly when will be linked with the API call

Reply:
eg: PENDING/EXISTS and other values will be dispatched by the related saga